opensource/project/software-req-win.md

```diff
@@ -65,6 +97,43 @@ To test it, stop the system Docker daemon and start the one you just built:
 
 The other make targets work too, to run unit tests try: `docker run --rm docker-builder sh -c 'cd /c/go/src/github.com/moby/moby; hack/make.sh test-unit'`.
 
+### 6. (Optional) Remove the interim container holding the built executable binaries:
+
+    docker rm binaries
+    
+### 7. (Optional) Remove the image used for the container in which the executable binaries are build. 
```

Comment:
These titles are too long for headings. Consider not using H3-level headings, but using list items with bold initial sentences.